site/snippets/templates/plugins/categories/seo.php

```diff
@@ -8,6 +8,7 @@
 	'plugins' => [
 		'plugins/diesdasdigital/meta-knight',
 		'plugins/fabianmichael/meta',
+		'plugins/tobimori/seo',
```

Comment:
Please move your plugin to the first position - it makes sense to feature the v4 compatible plugin more.